src/vs/workbench/services/textfile/common/textFileEditorModel.ts

```diff
@@ -321,12 +321,21 @@ export class TextFileEditorModel extends BaseTextEditorModel implements ITextFil
 			etag = this.lastResolvedDiskStat.etag; // otherwise respect etag to support caching
 		}
 
+		this.taskNotification();
```

Comment:
Maybe this should only happen after opening the file succeeds? Partly to avoid slowing down the file open with extra work, but also, a file can fail to open, in which case you don't want to show the notification.